nodejs/eks/cluster.ts

```diff
@@ -80,6 +80,29 @@ export interface CoreData {
     vpcCni?: VpcCni;
 }
 
+function translateAWSProviderToIAMAuth(clusterName: string, providerProfile?: string, providerAssumeRole?: { roleArn?: string }): pulumi.Output<any> {
+    const command = "aws-iam-authenticator";
+    const args = ["token", "-i", clusterName];
+    return pulumi.all([providerProfile, providerAssumeRole]).apply(([profile, assumeRole]) => {
```

Comment:
The `pulumi.all` should not be needed here - because the arguments passed to it are not `Output`s. You should be able to use the values of `profile` and `assumeRole` directly.

Reply:
Fixed